shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui25/components/KCText.kt

```diff
@@ -0,0 +1,23 @@
+package org.jetbrains.kotlinconf.ui25.components
+
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import org.jetbrains.kotlinconf.ui25.theme.KotlinConfTheme
+
+@Composable
+fun KCText(
```

Comment:
Could you tell me if this is a name from the design? If not, I would try naming it by the place it will be used

Reply:
This doesn't exist as a component in the design itself, it's only here to ease the implementation. Since we don't want to use Material `Text` directly, we need to drop down to `BasicText` instead, which for some reason takes a `ColorProducer` function as a parameter instead of a simple `Color` value.

So this component helps us create text and pass in colour to it easily, while also defaulting to the most common look of `KotlinConfTheme`. We can certainly add some documentation on the component.

Not sure what to name it though. Ideally, we'd use the `Text` name for it (as our design system substitues Material), but that only really works once we've removed the dependency on Material, otherwise it'll be a headache to import the correct one.

Reply:
We can try something like `BaseText` or `StyledText`

Reply:
Renamed to `StyledText` and added a comment to explain it